File: java/src/main/java/za/co/entelect/challenge/command/ChangeLaneCommand.java
package za.co.entelect.challenge.command;

import za.co.entelect.challenge.enums.Direction;

public class ChangeLaneCommand implements Command {

    private Direction direction;

    public ChangeLaneCommand(int laneIndicator) {
        if (laneIndicator == 1) {
            this.direction = Direction.valueOf("RIGHT");
        } else {
            this.direction = Direction.valueOf("LEFT");
        }
    }

    @Override
    public String render() {
        return String.format("TURN_%s", direction.getLabel());
    }
}

File: dotnetcore/StarterBot/Entities/Commands/ChangeLaneCommand.cs
using StarterBot.Enums;

namespace StarterBot.Entities.Commands
{
    public class ChangeLaneCommand: ICommand
    {
       public string direction;

       public ChangeLaneCommand(int laneIndicator) {
            if (laneIndicator == 1)
            {
                this.direction ="RIGHT";
            }
            else
            {
                this.direction = "LEFT";
            }
        }


        public string RenderCommand()
        {
            return $"TURN_{direction}";
        }
    }
}
